plugin/arcconf/arcconf.py

```diff
@@ -543,44 +543,45 @@ def disks(self, search_key=None, search_value=None,
         return search_property(rc_lsm_disks, search_key, search_value)
 
     @_handle_errors
-    def volume_raid_create(self, name, raid_type, disks, strip_size,
-                           flags=Client.FLAG_RSVD):
+    def volume_raid_create(self, name, raid_type, disks, strip_size, flags=Client.FLAG_RSVD):
```

Comment:
my hard wrap setting in my IDE was set to 120 instead of 80 as per pep8. i missed notice of that and made changes. i have changed these text width in new commit